### PR TITLE
#827: Use short form imports in pubsub docs

### PR DESCRIPTION
**Pull request by tseaver**
Uses #847 as a base.

Per #827.

@jgeewax, PTAL (particuarly at 4e1d2c9).

### CODE REVIEW

**Comment by dhermes** [comment]
Does this have another PR as a diffbase?

Overall LGTM.

I noticed you added a forgotten method to `pubsub-usage`. Any reason we don't just use the Sphinx autodoc features so we don't have to worry about stuff like that?

File: docs/pubsub-usage.rst [hunk 1]
```diff
@@ -45,16 +45,16 @@ List topics for the default project:
 
 .. doctest::
 
-   >>> from gcloud import pubsub
-   >>> [topic.name for topic in pubsub.list_topics()]  # API request
+   >>> from gcloud.pubsub import list_topics
```
Comment (jgeewax) [inline docs/pubsub-usage.rst]: Any reason why we switched these? Don't think it's a big deal but just curious.

**Comment by jgeewax** [comment]
LGTM

File: docs/pubsub-usage.rst (hunk 1 above)
Comment (tseaver) [inline docs/pubsub-usage.rst]: Consistency.  If we import other things as `from gcloud.pubsub import Foo`, then we should import the functions that way too.

**Comment by tseaver** [comment]
> Does this have another PR as a diffbase?

#847 (the original description says so.

> Overall LGTM.

> I noticed you added a forgotten method to pubsub-usage. Any reason we don't just use the Sphinx
> autodoc features so we don't have to worry about stuff like that?

This isn't API reference documentation -- it is "common pattern" / usage docs.

**Pull request merged**